application/features/VNC.py

```diff
@@ -188,4 +198,4 @@ def create_tunnel(self, remote_port):
                                         args=[local_websocket_port, local_vnc_port])
         proxy_thread.start()
 
-        return local_websocket_port
+        return local_websocket_port
```

Comment:
Let's avoid removing EOLs at EOFs.
```suggestion
        return local_websocket_port

```